salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -6,13 +6,24 @@
 {%- set roles = pillar.get('metalk8s', {}).get('nodes', {}).get(node_name, {}).get('roles', []) %}
 
 {%- if node_name not in salt.saltutil.runner('manage.up') %}
+# Salt-ssh need python3 to be installed on the destination host, so install it
+# manually using raw ssh
+Install python36:
+  metalk8s.saltutil_cmd:
+    - name: '[ "$EUID" -eq 0 ] && yum install -y python3 || sudo yum install -y python3'
+    - tgt: {{ node_name }}
+    - ssh: true
+    - raw_shell: true
+    - roster: kubernetes
+
 Deploy salt-minion on a new node:
   salt.state:
     - ssh: true
     - roster: kubernetes
     - tgt: {{ node_name }}
     - saltenv: metalk8s-{{ version }}
     - sls:
+      - metalk8s.salt.minion.dependencies
```

Comment:
Shouldn't this be `include`d in `roles.minion` instead? I get that we need to run it first to avoid failing the installation of other packages, but can't we handle this ordering with `include`s?

Reply:
Nop we can't because if we start the salt process without this `python36-rpm` installed once it come to salt-minion installation it will not compare the package version as expected (even if python36-rpm get installed during this state)